migrations/sql/V2020.02.27.1616__RemoveReportRoles.sql

```diff
@@ -1,11 +1,14 @@
 /* the previous migration has a mis-spelling of the role names so nothing got deleted */
 DO $$
     declare reader_id int;
+    declare scoped_reader_id int;
     declare reviewer_id int;
 begin
-    SELECT id INTO reader_id from role where name = 'reports-reader';
+    SELECT id INTO reader_id from role where name = 'reports-reader' is null;
+    SELECT id INTO scoped_reader_id from role where name = 'reports-reader' is not null;
```

Comment:
Should these be 'and scope_id is null' (or is not null?)
Is there definitely only one scoped reader role? Not one for report and one for report version?

Reply:
Oh, it's not scope-id is it? Scope_prefix?

Reply:
er yes :woman_facepalming: